engine/src/org/pentaho/di/trans/steps/sort/SortRowsMeta.java

```diff
@@ -320,7 +341,8 @@ public void getFields( RowMetaInterface inputRowMeta, String name, RowMetaInterf
         ValueMetaInterface valueMeta = inputRowMeta.getValueMeta( idx );
         valueMeta.setSortedDescending( !ascending[i] );
         valueMeta.setCaseInsensitive( !caseSensitive[i] );
-
+        valueMeta.setCollatorDisabled( !collatorEnabled[i] );
```

Comment:
Ok, i was following the existing convention, SortAcending vs Decending, CaseInsensitve vs Sensitive. Maybe i should change those as well?